mariadb/stack.yml

```diff
@@ -1,16 +1,15 @@
 # Use root/example as user/password credentials
-version: '3.1'
 
 services:
 
   db:
-    image: mariadb
+    image: mariadb:11.7
     restart: always
     environment:
       MARIADB_ROOT_PASSWORD: example
 
   adminer:
-    image: adminer
+    image: adminer:4.17.1
```

Comment:
Adminer upstream has been [very active recently](https://github.com/vrana/adminer/releases/) with many new version releases added to [Docker Official Images](https://github.com/docker-library/official-images/pulls?q=is%3Apr+label%3Alibrary%2Fadminer+is%3Aclosed). I don't know how long the very specific `4.17.1` image will continue to be "maintained". While we will be very unlikely to delete the tag from Docker Hub, this version will bit rot fairly quickly. So, I suggest keeping this as `latest`.